src/cswinrt/code_writers.h

```diff
@@ -6371,6 +6410,36 @@ private struct InterfaceTag<I>{};
             type_name,
             bind([&](writer& w)
             {
+                if (is_fast_abi_class(type))
+                {
+                    int hierarchy_index = get_class_hierarchy_index(type);
+                    if (!type.Flags().Sealed() || hierarchy_index > 0)
+                    {
+                        auto default_interface = get_default_interface(type);
+                        auto default_iface_method_count = 0;
+                        for_typedef(w, get_type_semantics(default_interface), [&](TypeDef iface)
+                            {
+                                default_iface_method_count = distance(iface.MethodList());
+                            });
+
+                        w.write(R"(
+protected unsafe % List<Func<IObjectReference>> GetDefaultFactories()
```

Comment:
Given that this function is returning a list of accessors to the fastabi-flattened default interfaces for the composition hierarchy, and not a list of factories to create them, a better name might be GetDefaultInterfaceAccessors or just GetDefaultInterfaces